### PR TITLE
Delete obsolete SIMD.js support

### PR DESCRIPTION
**Pull request by juj**
Delete obsolete SIMD.js support.

### CODE REVIEW

**Comment by juj** [comment]
Re-requesting reviews: expanded this PR to drop the SIMD.js paths. Also removed -s SIMD=1 setting, now just -msimd128 remains.

File: src/settings_internal.js [hunk 1]
```diff
@@ -28,6 +28,9 @@ var SYSCALLS_REQUIRE_FILESYSTEM = 1;
 // the features section.
 var BINARYEN_FEATURES = [];
 
+// Tracks whether to enable Wasm SIMD support in Binaryen.
+var BINARYEN_SIMD = 0;
```
Comment (kripken) [inline src/settings_internal.js:32]: Why is this needed? I think it should work automatically with `BINARYEN_FEATURES` like other wasm features. Maybe I'm forgetting something @tlively ?

Comment (juj) [reply src/settings_internal.js:32]: It does not seem to. Not sure if it's not hooked up, or?

**Comment by juj** [comment]
> About the test removals, is it not worth converting some of them for the new simd?

It might be, but I don't have the cycles to research that. There do already exist wasm simd tests, not sure how complete those are.

File: src/settings_internal.js (hunk 1 above)
Comment (kripken) [reply src/settings_internal.js:32]: That could be a bug in LLVM then if it isn't writing out the simd feature @tlively ?

Comment (tlively) [reply src/settings_internal.js:32]: Yeah this should Just Work™️. I can look into it as a follow-up.

**Comment by ngzhian** [comment]
@tlively this doc should be updated too? https://docs.google.com/document/d/1oy_bYCtMBIqhxhPTVpWgQFygazKL2FVwILaf3BrcacQ/edit
Or maybe that doc is out of date and not meant to be followed :) I just had it bookmarked an always referred to it.

**Comment by tlively** [comment]
I think that doc has probably been superseded by the official Emscripten documentation, but I'll update it to point to the right place.
